lib/staging/tls/tests/README.md

```diff
@@ -0,0 +1,41 @@
+# Tests
+
+Building tests:
+
+```
+$ cd everest-core
+$ mkdir build
+$ cd build
+$ cmake -GNinja -DEVEREST_CORE_BUILD_TESTING=ON ..
+$ ninja install
+```
+
+`touch release.json` may be needed if it hasn't been created (then re-run `ninja install`).
+
+## Unit tests
+
+- `./openssl_test`
```

Comment:
Is this up to date? Because I can not find this

Reply:
updated.